packages/flutter/lib/src/widgets/interactive_viewer.dart

```diff
@@ -147,6 +149,9 @@ class InteractiveViewer extends StatefulWidget {
        constrained = false,
        child = null;
 
+  /// Transform the child by applying a transformation matrix.
```

Comment:
```suggestion
  /// Transform the child by applying a transformation matrix.
  ///
  /// Defaults to true.
```

Reply:
Can we add some more doc here that explains what happens when this parameter is true or false?
Explaining what it does to hit tests etc like you did in the description of the PR seems like a good start?

Reply:
Will do!